### PR TITLE
update swipe tabs feedback

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/0/1206668020504729/f
Tech Design URL: 
CC:

**Description**:
Makes swipe tabs available to all users.

**Steps to test this PR**:
1. Add multiple tabs and swipe the navigation bar to navigate between them.
2. Swiping beyond the last tab should open a new tab if there wasn't there already.
3. Open and close tabs from the tab switcher and check everything updates correctly.

### CODE REVIEW

**Review by samsymons — APPROVED** [review]
LGTM, I noticed one issue during testing but I don't think it's a blocker, just a bit awkward. When you rotate your device then scroll over to a blank tab, the Dax icon is huge (but fixes itself when you finish scrolling).

https://github.com/duckduckgo/iOS/assets/183774/291d88a0-f468-4675-a840-6ebc06948244

File: DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -140,8 +140,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/apple/swift-argument-parser",
       "state" : {
-        "revision" : "46989693916f56d1186bd59ac15124caef896560",
-        "version" : "1.3.1"
+        "revision" : "c8ed701b513cf5177118a175d85fbbbcd707ab41",
+        "version" : "1.3.0"
```
Comment (samsymons) [inline DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved]: Feel like I've seen this a couple times lately. Wonder if one of our upstream dependencies isn't using a fixed version? Should probably reset this to the version on `main`.

Comment (brindy) [reply DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved]: I'll update from main again before I merge :)

Comment (brindy) [reply DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved]: Had to tell exclude to explicitly update it :/

**Pull request merged**